CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.33.0-rc.1 / 2022-01-19
+
+* [ENHANCEMENT] UI: Debounce search bar. #10157
+* [BUGFIX] TSDB: Fix deadlock from simultaneous GC and write. #10166
```

Comment:
Can it only happen in agent mode? It sounded to me it could also happen in normal mode, but if I misunderstood, this should be labeled "agent" for sure.